hydra/controllers/client.py

```diff
@@ -816,3 +816,149 @@ def disable_metrics(self):
 
         self.restart_service()
         self.app.log.info(f'Successfully disabled all metrics reporting.')
+
+    @ex(
+        arguments=[
+            (
+                ['-n', '--name'],
+                {
+                    'help': 'name of network for key backup',
+                    'action': 'store',
+                    'dest': 'name'
+                }
+            ),
+            (
+                ['--d', '--destination'],
+                {
+                    'help': 'directory to which keys are backed up',
+                    'action': 'store',
+                    'dest': 'destination'
+                }
+            ),
+            (
+                ['-D', '--destroy'],
+                {
+                    'help': 'Use to overwrite existing backups',
+                    'action': 'store_true',
+                    'dest': 'destroy'
+                }
+            ),
+        ]
+    )
+    def backup_nodekeys(self, destroy=False):
+        name = self.app.utils.env_or_arg(
+            'name', 'HYDRA_NETWORK', or_path='.hydra_network', required=True)
+
+        filenames = [f'chaindata/config/node_key.json',
+                     f'chaindata/config/priv_validator.json']
+
+        destination = self.app.utils.path(self.app.pargs.destination) if self.app.pargs.destination \
+            else os.environ['HOME'] + '/.hydra'
+
+        if not os.path.exists(destination):
+            os.makedirs(destination)
+
+        project_path = os.getcwd()
+
+        for filename in filenames:
+            fn = filename.split('/')[-1] #filename.json
+            directory_tree = name + '/' + filename.rsplit('/', 1)[0] #{name}/path/to/directory
```

Comment:
These lines could be simplified to
`directory_tree, filename = path.rsplit('/', 1)`
`dest_path = f'{name}/{directory_tree}/{filename}`